src/main/kotlin/org/rekotlin/Action.kt

```diff
@@ -28,10 +25,10 @@ package org.rekotlin
  * All actions that want to be able to be dispatched to a store need to conform to this protocol
  * Currently it is just a marker protocol with no requirements.
  */
-interface Action {}
+interface Action
```

Comment:
README.md needs to be updated explaining 2 possibilities
1. Use Block subscriber
2. Use delegate class that subscribe to the state, use the delegate class as a member, subscribing and unsubscribing as per the lifecycle needs

Reply:
Updated README for now. Though i believe one of the samples need to be updated. I'll look into that.